app/js/components/Home/Index.react.js

```diff
@@ -106,4 +106,4 @@ function mapStateToProps(state) {
   };
 }
 
-export default connect(mapStateToProps)(Home);
+export default connect(mapStateToProps)(Index);
```

Comment:
What does this component do?
Why doesn't it have a render method?
Why is it not a functional component. where is it's template